pkg/sentry/platform/ring0/entry_arm64.s

```diff
@@ -300,17 +300,25 @@
 
 // SWITCH_TO_APP_PAGETABLE sets a new pagetable for a container application.
 #define SWITCH_TO_APP_PAGETABLE(from) \
-	MOVD CPU_TTBR0_APP(from), RSV_REG; \
-	WORD $0xd5182012; \	//        MSR R18, TTBR0_EL1
 	ISB $15; \
-	DSB $15;
+	MRS TTBR1_EL1, R0; \
+	MOVD CPU_APP_ASID(from), R1; \
+	BFI $48, R1, $16, R0; \
+	MSR R0, TTBR1_EL1; \
+	ISB $15; \
```

Comment:
Can you clarify why two ISBs are needed in this macro (and `SWITCH_TO_KVM_PAGETABLE`), before both the stores to TTBR1_EL1 and TTBR0_EL1 respectively?

Reply:
Hi @nixprime 
I have removed the ISB before the store.
I think one ISB after  'MSR R0, TTBR1_EL1' is enough to ensure ensure that possible change is visible by the core.
Thanks for your reminder.